mesa/examples/basic/boltzmann_wealth_model/Readme.md

```diff
@@ -2,18 +2,18 @@
 
 ## Summary
 
-A simple model of agents exchanging wealth. All agents start with the same amount of money. Every step, each agent with one unit of money or more gives one unit of wealth to another random agent. This is the model described in the [Intro Tutorial](https://mesa.readthedocs.io/en/latest/tutorials/intro_tutorial.html), with the completed code.
+A simple model of agents exchanging wealth. All agents start with the same amount of money. Every step, each agent with one unit of money or more gives one unit of wealth to another random agent. This is the model described in the [Intro Tutorial](https://mesa.readthedocs.io/latest/tutorials/0_first_model.html), with the completed code.
```

Comment:
This write up doesn't make a lot of sense now that the intro tutorial is a series. What if we change it from "This model is described..." to something like...

"Mesa's [Getting Started](https://mesa.readthedocs.io/latest/getting_started.html) section walks through the Boltzmann Wealth Model in a series of short introductory tutorials, starting with [Creating your First Model](https://mesa.readthedocs.io/latest/tutorials/0_first_model.html).